### PR TITLE
[Topic Entry] Python: Pillow

### PR DESCRIPTION
**Pull request by mamtawardhani**
### Description

Added a topic entry for the Pillow Library in Python.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [xx] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by Sriparno08 — APPROVED** [review]
Looks good for a second review, @mamtawardhani! 👍

File: content/pillow/pillow.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+In Python, the [**Pillow**](https://pypi.org/project/pillow/) library is a powerful tool used for [image processing](https://www.codecademy.com/article/getting-started-with-image-processing-in-python-using-pillow) and manipulation, serving as an enhanced version of the Python Imaging Library (PIL). It not only supports various image formats like JPEG, PNG, GIF, TIFF, and BMP, but also offers capabilities such as resizing, cropping, rotating, and color adjustments. Pillow allows for applying filters, adding text and shapes, and performing image analysis tasks. Its ease of use makes it suitable for both beginners and experienced developers. With active community support and extensive documentation, Pillow is a versatile choice for a wide range of image processing applications.
+
+Pillow consists of a lot of modules, each catering to different aspects of image processing and manipulation in Python.
+
+## Installation
+
+The official release of Pillow can be installed with [pip](https://www.codecademy.com/resources/docs/python/pip):
+
+```py
```
Comment (letsbuildanything) [inline content/pillow/pillow.md:18]:
```suggestion
```shell
```

**Review by letsbuildanything — CHANGES_REQUESTED** [review]
@mamtawardhani suggested and made some changes. Please take a moment to review them.
Thank you.

**Comment by mamtawardhani** [comment]
@letsbuildanything I've made the required changes! 💯

**Review by letsbuildanything — APPROVED** [review]
Ready to Merge! 🚀

**Pull request merged**